Challenge-4/Bot_1/README.md

```diff
@@ -14,19 +14,19 @@ The bot monitors different governance values that can change if a proposal is su
 Describe each of the type of alerts fired by this agent
 
 - GOV-1
-  - Fired when the any of the governance proposal is changed in Compound V3 protocol i.e. `SetSupplyKink, setBorrowKink, setGovernor, setBorrowPerYearInterestRateBase, setSupplyPerYearInterestRateBase, UpdateAssetLiquidateCollateralFactor, UpdateAssetBorrowCollateralFactor`
+  - Fired when the any of the governance proposal is changed in Compound V3 protocol e.g. `SetSupplyKink`, `setBorrowKink`, `setGovernor`, `setBorrowPerYearInterestRateBase`, `setSupplyPerYearInterestRateBase`, `UpdateAssetLiquidateCollateralFactor`, `UpdateAssetBorrowCollateralFactor`
   - Severity is always set to `Info` as it detects changes made to the protocol which do not involve any security risk.
   - Type is always set to `Info`
   - Metadata fields: 
-    - `changedEvents`: List of events that are changed in the proposal:
+    - Changed Events: List of events that are changed in the proposal:
       - `SUPPLY_KINK`: Old and new values only if Supply Kink is changed
       - `BORROW_KINK`: Old and new values only if Borrow Kink is changed
       - `SET_GOVERNOR`: Old and new values only if Governor is changed
       - `BORROW_PYIR`: Old and new values only if Borrow Per Year Interest Rate is changed
       - `SUPPLY_PYIR`: Old and new values only if Supply Per Year Interest Rate is changed
       - `LIQUIDATE_CF`: Old and new values only if Asset Liquidate Collateral Factor is changed
       - `BORROW_CF`: Old and new values only if Asset Borrow Collateral Factor is changed
-      - Asset Data: Each token has a different Asset data. For example USDC token on Ethereum: `"Collateral Asset - Compound": "0xc00e94Cb662C3520282E6f5717214004A7f26888",
+    - Asset Data: Each token has a different Asset data. For example USDC token on Ethereum: `"Collateral Asset - Compound": "0xc00e94Cb662C3520282E6f5717214004A7f26888",
     "Collateral Asset - Wrapped BTC": "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599",
     "Collateral Asset - Wrapped Ether": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
     "Collateral Asset - Uniswap": "0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984",
```

Comment:
```suggestion
    - Asset Data: Each token has a different Asset data. For example USDC token on Ethereum:
      - `"Collateral Asset - Compound": "0xc00e94Cb662C3520282E6f5717214004A7f26888"`,
      - `"Collateral Asset - Wrapped BTC": "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599"`,
      - `"Collateral Asset - Wrapped Ether": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"`,
      - `"Collateral Asset - Uniswap": "0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984"`
```
Just a small suggestion to make it more legible. Though, I thought I had left a comment previously that it could also just be a format of what someone could expect. Such as `"Collateral Asset - <asset name>": "<asset address>"`.